test/utils/fizzy_engine.cpp

```diff
@@ -24,6 +25,19 @@ class FizzyEngine : public WasmEngine
     Result execute(FuncRef func_ref, const std::vector<uint64_t>& args) final;
 };
 
+namespace
+{
+// fizzy::execution_result host_mul256(fizzy::Instance&, std::vector<uint64_t>, int)
+//{
+//    return {false, {}};
+//}
+fizzy::execution_result host_crc32(fizzy::Instance& instance, std::vector<uint64_t> args, int)
+{
+    auto ret = fizzy::adler32(instance.memory->substr(args[0], args[1]));
```

Comment:
Doesn't `substr()` do a copy?

Reply:
Given `memory` is a `bytes_view`: https://en.cppreference.com/w/cpp/string/basic_string_view/substr